docs/pages/components/slider-styled.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
+import { prepareMarkdown } from 'docs/src/modules/utils/parseMarkdown';
+
+const pageFilename = 'components/slider-styled';
+const requireDemo = require.context(
+  'docs/src/pages/components/slider-styled',
+  false,
+  /\.(js|tsx)$/,
+);
+const requireRaw = require.context(
+  '!raw-loader!../../src/pages/components/slider-styled',
+  false,
+  /\.(js|md|tsx)$/,
+);
+
+// Run styled-components ref logic
+// https://github.com/styled-components/styled-components/pull/2998
+requireDemo.keys().map(requireDemo);
```

Comment:
I wouldn't be against mocking React.useRef and throw so we don't have to add this in each page. https://github.com/styled-components/styled-components/pull/2998/files#diff-25955a9ec09b01d6a45a389189fb0f6dR20 🙃

Reply:
Wouldn't work since we'd have to mock it for render time. Their logic is correct and they could just use any other hook.

In the it's better to do this know since we'll eventually have these modules evaluated before render anyway. It's a good smoke test to be sure we are compatible with how ES modules behave.